src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsRulesSpec.groovy

```diff
@@ -685,20 +683,20 @@ class PriceFloorsRulesSpec extends PriceFloorsBaseSpec {
         def account = getAccountWithEnabledFetch(bidRequest.site.publisher.id)
         accountDao.save(account)
 
-        and: "Set Floors Provider response without model weights defined"
+        and: "Set Floors Provider response with model weights defined"
         def floorValue = PBSUtils.randomFloorValue
         def floorsResponse = PriceFloorRules.priceFloorRules.tap {
             data.modelGroups << ModelGroup.modelGroup
             data.modelGroups[0].schema = new PriceFloorSchema(fields: [MEDIA_TYPE])
             data.modelGroups[0].values = [(new Rule(mediaType: BANNER).rule): floorValue]
-            data.modelGroups[0].modelWeight = null
+            data.modelGroups[0].modelWeight = 1
             data.modelGroups[1].schema = new PriceFloorSchema(fields: [DEVICE_TYPE])
             data.modelGroups[1].values =
                     [(new Rule(deviceType: PHONE).rule): floorValue + 0.1,
                      (new Rule(deviceType: TABLET).rule): floorValue + 0.2,
                      (new Rule(deviceType: DESKTOP).rule): floorValue + 0.3,
                      (new Rule(deviceType: MULTIPLE).rule): floorValue + 0.4]
-             data.modelGroups[1].modelWeight = null
+             data.modelGroups[1].modelWeight = 2
```

Comment:
Test should choose, this value (new Rule(deviceType: MULTIPLE).rule): floorValue + 0.4], on line 711